lib/logstash/outputs/scalyr.rb

```diff
@@ -284,14 +299,19 @@ def build_multi_event_request_array(logstash_events)
     thread_ids = Hash.new
     next_id = 1 #incrementing thread id for the session
 
+    # per-logfile attributes
```

Comment:
Are there test cases for the scalyr event / per log file attribute JSON serialization?

Reply:
The tests do check for serverHost and logfile in the logfile attributes, but I didn't add a test that would bring an event field into it actually, I'll add one.